bin/update.sh

```diff
@@ -84,12 +103,21 @@ checksum=$(shasum -a 256 Glean.xcframework.zip | awk '{print $1}')
 echo "Checksum: ${checksum}"
 
 FILE=Package.swift
+if [ $UPDATE_IOS == "y" ]; then
 run $SED -i.bak -E \
     -e "s/^let version = \"[0-9a-z.-]+\"/let version = \"${NEW_VERSION}\"/" \
     -e "s/^let checksum = \"[0-9a-z.-]+\"/let checksum = \"${checksum}\"/" \
+    -e "s/\.iOS\(\.v[0-9][0-9]\)/.iOS(.${IOS_VERSION})/" \
     "${WORKSPACE_ROOT}/${FILE}"
+else
+run $SED -i.bak -E \
+    -e "s/^let version = \"[0-9a-z.-]+\"/let version = \"${NEW_VERSION}\"/" \
+    -e "s/^let checksum = \"[0-9a-z.-]+\"/let checksum = \"${checksum}\"/" \
+    "${WORKSPACE_ROOT}/${FILE}"
+fi
```

Comment:
Maybe the additional replace should be on its own instead of duplication the previous replaces.
Additionally there's no need for a new variable, let's just test if the version is set: 

```suggestion
run $SED -i.bak -E \
    -e "s/^let version = \"[0-9a-z.-]+\"/let version = \"${NEW_VERSION}\"/" \
    -e "s/^let checksum = \"[0-9a-z.-]+\"/let checksum = \"${checksum}\"/" \
    "${WORKSPACE_ROOT}/${FILE}"
if [ -n "$IOS_VERSION" ]; then
run $SED -i.bak -E \
    -e "s/\.iOS\(\.v[0-9][0-9]\)/.iOS(.${IOS_VERSION})/" \
    "${WORKSPACE_ROOT}/${FILE}"
fi
```